src/gatsby-node.js

```diff
@@ -17,16 +17,27 @@ const headersWithToken = (headers, token) => {
   return headers;
 };
 
+// Helper to add expansions parameters
+const urlWithExpansions = (url, expansions) => {
+  if (expansions) {
+    return `${url}?expand=${expansions.join()}`;
```

Comment:
@ajayns Should that be `join(’,’)`?

Reply:
@datakurre the `join()` method by default joins a items in an array separated by commas, so `join()` and `join(',')` both returns the same thing. Although I could change as you suggested to make it more readable?